.eslintrc.js

```diff
@@ -379,7 +379,7 @@ module.exports = {
 		{
 			files: [
 				'**/@(storybook|stories)/*',
-				'packages/components/src/**/*.tsx',
```

Comment:
Nitty, but it would be great to move this to a separate PR to keep the scope of this one tight

Reply:
It definitely should be separate, since the omission of `ts` files was deliberate! We got some strong pushback on this point (#39458).

Reply:
I've only skimmed that issue, but is the main point about docgen? Because if that's the case I'd be happy to fix whatever needs to be fixed. This rule and its autofix almost caused my utilities to be wrongly named and documented, as it adds blank `@param` declarations, so the result would have been arguably worse. In any case, I'm just gonna disable it for this file since these utils are internal, and leave that matter for another day.

Thanks for the context!

Reply:
Reverted change.